_includes/_acquire.htm

```diff
@@ -13,25 +13,21 @@ <h4 class="text-bold">
             <div class="col-12">
                 <ul>
                     <li>
-                        Before your wallet can do anything, it needs to be synced (the green tick under connections means you are synced)
+                        Before your wallet can do anything, it needs to be synced. The cloud icon at the bottom right of the wallet shows your sync status, you can hover the mouse over it for detailed information. A green cloud means it is fully synced. Note that the first time you run the wallet, it may take several hours to fully sync depending on your computer and internet connection speed.
                     </li>
                     <li>
                         If you don't want to sync from 0, then you can download a snapshot of the blockchain from clicking File -> Snapshot Download or using the <code>-snapshotdownload</code> command line argument
                         <ul>
                             <li>
                                 The snapshot will only bring you up to 80-90% sync. Allow it to continue syncing after the snapshot has been extracted.
                             </li>
-                            <li>
-                                If you want to apply it directly, you can download the snapshot <a href="https://snapshot.gridcoin.us/snapshot.zip">here</a>.
```

Comment:
why is this being removed?

Reply:
Because:
- The wallet already provides this functionality
- It looks confusing as hell and will scare new users and is non-necessary information
- It only applies to users installing wallet on multiple machines or for some reason who already have the snapshot downloaded?

Maybe I am confused here, downloading it in the wallet doesn't require shutting down the wallet and moving files in-place right? It handles all that for you?

Reply:
Not everyone runs the GUI...

Reply:
Yes and we should provide new user documentation based on the majority of users. The majority of users will not have used a CLI. And we can expect the CLI users to be technically adept enough to research snapshots if they want to.

Reply:
If anything, the CLI setup should probably be an entirely different guide. This guide generally doesn't provide all the cli setup steps needed. For example, how to make a beacon.

Reply:
I guess actually at the final step there are the CLI instructions. Maybe this should be moved there then?

Reply:
What should be the final resolution of this commit?

Reply:
the non-GUI version of the wallet also allows managed snapshot download.

Reply:
-snapshotdownload
       Download and apply latest snapshot

  -snapshotsha256url=<url>
       Optional: URL for the snapshot.sha256 file (ex:
       https://sub.domain.com/location/snapshot.sha256)

  -snapshoturl=<url>
       Optional: URL for the snapshot.zip file (ex:
       https://sub.domain.com/location/snapshot.zip)

Reply:
In no instance is it preferable to manually download and install a snapshot.